src/status_im/ui/screens/about_app/styles.cljs

```diff
@@ -17,3 +17,13 @@
 
 (def about-title-text
   {:font-size 20})
+
+(def learn-more-title
+  {:font-size   17
```

Comment:
you can use typography instead

Reply:
Do you mean i can remove `line-height` and rely on automatic calculation, or something else?

Reply:
i mean we don't use styles for text anymore, only `:typography` because all labels are standatized so instead of `[react/text {:style}]` you should use `[react/text {:typography}`

Reply:
Thanks for clarifying, fixed.